src/main/java/com/iota/iri/conf/ProtocolConfig.java

```diff
@@ -7,34 +7,59 @@
 public interface ProtocolConfig extends Config {
 
     /**
-     * @return Descriptions#MWM
+     * Default Value: {@value BaseIotaConfig.Defaults#MWM}
+     *
+     * @return {@value ProtocolConfig.Descriptions#MWM}
      */
     int getMwm();
 
     /**
-     * @return Descriptions#TRANSACTION_PACKET_SIZE
+     * Default Value: {@value BaseIotaConfig.Defaults#PACKET_SIZE}
+     *
+     * @return {@value ProtocolConfig.Descriptions#TRANSACTION_PACKET_SIZE}
      */
     int getTransactionPacketSize();
 
     /**
-     * @return Descriptions#REQUEST_HASH_SIZE
+     * Default Value: {@value BaseIotaConfig.Defaults#REQ_HASH_SIZE}
+     *
+     * @return {@value ProtocolConfig.Descriptions#REQUEST_HASH_SIZE}
      */
     int getRequestHashSize();
 
     /**
-     * @return Descriptions#P_REPLY_RANDOM_TIP
+     * Default Value: {@value BaseIotaConfig.Defaults#P_REPLY_RANDOM_TIP}
+     *
+     * @return {@value ProtocolConfig.Descriptions#P_REPLY_RANDOM_TIP}
      */
     double getpReplyRandomTip();
 
+    /**
+     * Default Value: {@value BaseIotaConfig.Defaults#P_DROP_TRANSACTION}
+     *
+     * @return {@value ProtocolConfig.Descriptions#P_DROP_TRANSACTION}
+     */
     double getpDropTransaction();
 
     /**
-     * @return Descriptions#P_SELECT_MILESTONE
+     * Default Value: {@value BaseIotaConfig.Defaults#P_SELECT_MILESTONE_CHILD}
```

Comment:
the default value name does not match the description name.
`P_SELECT_MILESTONE_CHILD` vs `P_SELECT_MILESTONE`